### PR TITLE
Supervisor edit course

### PR DESCRIPTION
**Pull request by lethihaivan**
Screenshot from 2021-04-01 13-01-32

## Related Tickets
-(https://edu-redmine.sun-asterisk.vn/issues/35000)

## WHAT
-As an supervisor, I can edit (and customize structure of course) a course

## Evidence (Screenshot or Video)
Screenshot from 2021-04-01 13-05-29

### CODE REVIEW

**Comment by lethihaivan** [comment]
ready

File: app/views/supervisor/courses/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <%= f.label :time, t("courses.supervisor.new.time_estimate") %>
+    <%= f.text_field :time, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= f.label :status, t("courses.supervisor.new.status_coures") %>
+    <%= f.select(:status, load_coures_statuses) %>
+  </div>
+  <div class="form-group">
+    <%= f.label t("courses.supervisor.new.start_date")%>
+    <%= f.datetime_field :start_date,class: 'form-control'%>
```
Comment (vanvtt-0952) [inline app/views/supervisor/courses/_form.html.erb]: `'form-control'` nháy đôi nè 
và thêm sapce trước `%>`

File: app/views/supervisor/courses/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<% provide(:title, t("courses.supervisor.edit.title")) %>
+<section class="courses_area p_120">
+  <div class="container">
+    <h3 class="mb-30 title_color"><%= t "courses.supervisor.edit.title" %></h3>
+     <%= render "form", course: @course %>
```
Comment (vanvtt-0952) [inline app/views/supervisor/courses/edit.html.erb]: `, course: @course` -> xóa dc nè

File: app/views/supervisor/courses/new.html.erb [hunk 3]
```diff
@@ -2,37 +2,6 @@
...
-        <%= f.label t("courses.supervisor.new.subject")%>
-        <%= f.select(:subject_ids, load_subjects,
-          {prompt: t("courses.supervisor.new.select_subject")}, multiple: true) %>
-      </div>
-      <div class="form-group button_group">
-        <%= link_to t("courses.supervisor.new.cancel_button"), root_path, class: "btn btn-danger" %>
-        <%= f.submit t("courses.supervisor.new.create_button"), class: "btn btn-primary" %>
-      </div>
-    <% end %>
+      <%= render "form", course: @course %>
```
Comment (vanvtt-0952) [inline app/views/supervisor/courses/new.html.erb]: `, course: @course` xóa dc

**Comment by lethihaivan** [comment]
ready

**Pull request merged**